src/milia/api/io.cljs

```diff
@@ -64,11 +64,12 @@
               ["Accept" (or accept-header "application/json")]])))
 
 (defn get-xhr-io-response
-  [io-obj]
+  [io-obj & [{:keys [require-json?]}]]
```

Comment:
This changes the default behavior for existing code, shouldn't this default to true so the default behavior does not change. Also needs a doc string.